src/appkit.cs

```diff
@@ -896,12 +897,11 @@ interface NSApplicationDelegate {
 		[Mac (10,10, onlyOn64 : true)]
 		[Export ("application:didUpdateUserActivity:"), EventArgs ("NSApplicationUpdatedUserActivity"), DefaultValue (false)]
 		void UpdatedUserActivity (NSApplication application, NSUserActivity userActivity);
-#endif
 
-		//TODO - Waiting on CloudKit binding
-		//[Mac (10,12)]
-		//[Export ("application:userDidAcceptCloudKitShareWithMetadata:")]
-		//void UserDidAcceptedCloudKitShare (NSApplication application, CKShareMetadata metadata);
+		[Mac (10,12, onlyOn64 : true)]
+		[Export ("application:userDidAcceptCloudKitShareWithMetadata:"), EventArgs ("NSApplicationUserAcceptedCloudKitShare")]
+		void UserDidAcceptedCloudKitShare (NSApplication application, CKShareMetadata metadata);
```

Comment:
`UserDidAcceptedCloudKitShare` sounds weird to me (and is not the selector name) ?
`UserDidAcceptCloudKitShare` ?